src/libs/import.c

```diff
@@ -1135,6 +1135,18 @@ static gboolean _button_press(GtkWidget *view, GdkEventButton *event, dt_lib_mod
     }
     gtk_tree_path_free(path);
   }
+
+  if(event->type == GDK_DOUBLE_BUTTON_PRESS)
```

Comment:
This chunk of code is duplicate of the one above. So better create a function and call it to share the code.

Reply:
I only see one line  of code which is a duplicate. Is it worth it to create a new function for 1 line?

Reply:
No, indeed I missed the if() conditional which is not in both chunks.